src/controllers/product.controller.js

```diff
@@ -8,10 +8,27 @@ import { transformUserRoles } from '../helpers/transformUserRoles';
 import { verifyToken } from '../middleware/verifyToken';
 import dotenv from 'dotenv';
 import { Product } from '../database/models';
-import { expiryNot } from '../helpers/createotp'
+import { notifications } from '../helpers/sendNotification';
 
-const eventEmitter = new EventEmitter();
+const productEvents = new EventEmitter();
 
+// Function to send notifications
```

Comment:
@Emile-SHUMBUSHO, I had re-written most of this function in my previous PR. Maybe you could check it before making any change.